content/blog/2022/11/2022-11-24-jenkins-sponsor-appreciation.adoc

```diff
@@ -0,0 +1,49 @@
+---
+:layout: post
+:title: "Jenkins Sponsor Appreciation"
+:tags:
+- community
+- sponsors
+- governance
+authors:
+- kmartens27
+description: >
+    The Jenkins project shares gratitude to our amazing sponsors.
+---
+
+The Jenkins project would like to take a moment and share deep gratitude to our sponsors.
+Sponsorship of the Jenkins project is key to the maintenance and development of Jenkins.
+There are different ways in which our sponsor organizations support the Jenkins project, such as infrastructure, necessary development tools, funding, and providing mirrors to connect Jenkins to the community.
+ 
+For supporting Jenkins distribution, build, test, and deployment infrastructure, we would like to thank link:https://jfrog.com/[JFrog], https://aws.amazon.com/[AWS], link:https://www.oracle.com/[Oracle], link:https://www.digitalocean.com/[Digital Ocean], and link:https://osuosl.org/[OSU Open Source Labs].
+
+image:/images/sponsors/jfrog.png[Jfrog, width=15%]
+image:/images/sponsors/aws.png[AWS, width=25%]
+image:/images/sponsors/oracle.png[Oracle, width=35%]
+image:/images/sponsors/digital-ocean.png[Digital Ocean, width=30%]
+image:/images/sponsors/osuosl.png[OSU Open Source Labs, width=30%]
+
+We also want to recognize and thank link:https://github.com/[Github], link:https://jfrog.com/[JFrog], link:https://www.atlassian.com/[Atlassian], link:https://www.linuxfoundation.org/[Linux Foundation], link:https://www.netlify.com/[Netlify], and link:https://1password.com/[1Password] for providing tools to track Jenkins development.
+
+image:/images/sponsors/github.png[Github, width=30%]
+image:/images/sponsors/jfrog.png[Jfrog, width=15%]
+image:/images/sponsors/atlassian.png[Atlassian, width=40%]
+image:/images/sponsors/1password.png[1 Password, width=30%]
+image:/images/sponsors/netlify.png[Netlify, width=30%]
+image:/images/sponsors/linux-foundation.png[The Linux Foundation, width=27%]
+
+Massive thanks to link:https://www.algolia.com/[Algolia] for providing site search for the primary Jenkins and plugins sites.
```

Comment:
The phrasing felt odd to me.  Would an extra word help here?
```suggestion
Massive thanks to link:https://www.algolia.com/[Algolia] for providing site search for the primary Jenkins documentation and plugins sites.
```

Reply:
yes, definitely thanks @MarkEWaite!